src/module.json

```diff
@@ -13,14 +13,14 @@
       "url": "https://www.patreon.com/mclemente"
     }
   ],
-  "version": "0.3.0",
+  "version": "0.3.17",
   "minimumCoreVersion": "0.8.6",
   "compatibleCoreVersion": "0.8.9",
   "url": "https://github.com/Forien/foundryvtt-forien-unidentified-items",
-  "manifest": "https://raw.githubusercontent.com/League-of-Foundry-Developers/foundryvtt-forien-unidentified-items/master/module.json",
-  "download": "https://github.com/League-of-Foundry-Developers/foundryvtt-forien-unidentified-items/releases/download/v0.3.0/v0.3.0.zip",
-  "readme": "https://github.com/League-of-Foundry-Developers/foundryvtt-forien-unidentified-items/blob/v0.3.0/README.md",
-  "changelog": "https://github.com/League-of-Foundry-Developers/foundryvtt-forien-unidentified-items/blob/v0.3.0/changelog.md",
+  "manifest": "https://github.com/League-of-Foundry-Developers/foundryvtt-forien-unidentified-items/releases/latest/download/module.json",
```

Comment:
This field is overwritten by the release autopublish github action. We typically prefer to leave this blank so that there isn't any confusion about that